frontend/packages/gitops-plugin/src/components/details/GitOpsDetails.tsx

```diff
@@ -38,63 +97,112 @@ const GitOpsDetails: React.FC<GitOpsDetailsProps> = ({ envs, appName }) => {
-                        </span>
-                      </StackItem>
-                      {env.environment && argocdLink && (
-                        <StackItem className="co-truncate co-nowrap">
-                          <ExternalLink
-                            href={`${argocdLink.spec.href}/applications/${env.environment}-${appName}`}
-                            text="Argo CD"
-                            additionalClassName="odc-gitops-details__env-section__argocd-link"
-                          />
+                      {env.status && (
+                        <StackItem className="odc-gitops-details__env-section__status-label">
+                          <Tooltip content="Sync status">
+                            <span>{renderStatusLabel(env.status)}</span>
+                          </Tooltip>
+                        </StackItem>
+                      )}
+                    </Stack>
+                  </CardTitle>
+                  <CardBody>
+                    <Stack className="odc-gitops-details__revision">
+                      {env.revision ? (
+                        <>
+                          {env.revision?.message && (
+                            <StackItem className="odc-gitops-details__revision__message">
+                              {t('gitops-plugin~{{message}}', { message: env.revision.message })}
+                            </StackItem>
+                          )}
+                          <StackItem className="odc-gitops-details__revision__author-sha">
+                            {env.revision?.author && (
+                              <span className="odc-gitops-details__revision__author">
+                                {t('gitops-plugin~by {{author}}', { author: env.revision.author })}{' '}
+                              </span>
+                            )}
+                            {env.revision?.revision && (
+                              <Label
+                                className="odc-gitops-details__revision__sha"
+                                color="blue"
+                                icon={routeDecoratorIcon(
+                                  'https://www.github.com/reginapizza/taxi',
+                                  12,
+                                  t,
+                                )}
+                                variant="outline"
+                              >
+                                {env.revision.revision.substring(0, 7)}
+                              </Label>
+                            )}
+                          </StackItem>
+                        </>
+                      ) : (
+                        <span>{t('gitops-plugin~Commit details not available')}</span>
+                      )}
+                      {env.lastDeployed && (
+                        <StackItem className="co-truncate co-nowrap odc-gitops-details__env-section__time">
+                          {t('gitops-plugin~Last deployed')}&nbsp;{' '}
```

Comment:
```suggestion
                          {t('gitops-plugin~Last deployed')}{' '}
```
You don't need to use `nbsp` typically. Simply adding a space is usually good enough -- unless this kinda of instance happens, where two components are separated by a line break. That's really the only time, and you can use `{' '}` to reserve that space between the elements.